packages/hardhat/README.md

```diff
@@ -17,5 +17,5 @@ npx hardhat ignition deploy ./ignition/modules/Lock.ts
 For contracts deployed on Filecoin mainnet or Filecoin testnet (calibration) using this repo you can verify them by executing this script:
 
 ```shell
-npx hardhat verify-contract --network $networkName --contract-name $contractName
+npx hardhat verify-contract --network $networkName --contractname $contractName
```

Comment:
@nijoe1 using `--contract-name` is better practice here and it works with using `contractName` in deployScript, lets keep that only here.

Reply:
Yes but this is not working in the terminal that's why I did it like that

Reply:
Fixed @lordshashank 

Feel free to merge @xBalbinus